sysdump/sysdump.go

```diff
@@ -1036,13 +1036,7 @@ func (c *Collector) submitBugtoolTasks(ctx context.Context, pods []*corev1.Pod,
 			if len(m) != 2 || len(m[1]) == 0 {
 				return fmt.Errorf("failed to collect 'cilium-bugtool' output for %q in namespace %q: output doesn't contain archive name: %s", p.Name, p.Namespace, outString)
 			}
-
-			// Gzip bugtool tar file to reduce the size of transferred file
-			o, e, err = c.Client.ExecInPodWithStderr(ctx, p.Namespace, p.Name, containerName, []string{"gzip", m[1]})
-			if err != nil {
-				return fmt.Errorf("failed to compress 'cilium-bugtool' output for %q in namespace %q: %w: %s", p.Name, p.Namespace, err, e.String())
-			}
-			tarGzFile := m[1] + ".gz"
+			tarGzFile := m[1]
```

Comment:
`cilium sysdump` may be used on Cilium versions v1.9 to v1.12. Does cilium-bugtool support generating `.tar.gz` files on all those versions?

Reply:
It looks like it does. PR cilium/cilium#4264 added support for `--archiveType=gz` to the bugtool command. That flag is used here and produces a `.tar.gz` file. The respective commit seems to be present in all release branches from v1.9 to v1.12.

Reply:
Ah, perfect! Yeah, from 4y ago, so should be everywhere.

Reply:
I must confess, I assumed 4 years was old enough :-). Thank you indeed for checking!